ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/deserialize/ClaimDataMixIn.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.cmc.ccd.deserialize;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonUnwrapped;
+import uk.gov.hmcts.cmc.domain.models.Interest;
+import uk.gov.hmcts.cmc.domain.models.InterestBreakdown;
+import uk.gov.hmcts.cmc.domain.models.InterestDate;
+import uk.gov.hmcts.cmc.domain.models.Payment;
+import uk.gov.hmcts.cmc.domain.models.Timeline;
+import uk.gov.hmcts.cmc.domain.models.amount.Amount;
+import uk.gov.hmcts.cmc.domain.models.evidence.Evidence;
+import uk.gov.hmcts.cmc.domain.models.legalrep.StatementOfTruth;
+import uk.gov.hmcts.cmc.domain.models.otherparty.TheirDetails;
+import uk.gov.hmcts.cmc.domain.models.party.Party;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+
+public abstract class ClaimDataMixIn {
+
+    @JsonProperty("listOfClaimants")
```

Comment:
Can we have just `claimants` and `defendants`?  Other collections does not have prefix like `listOf...`.